src/couch_mrview/src/couch_mrview_index.erl

```diff
@@ -127,6 +127,12 @@ open(Db, State0) ->
                     NewSt = couch_mrview_util:init_state(Db, Fd, State, Header),
                     ensure_local_purge_doc(Db, NewSt),
                     {ok, NewSt};
+                {ok, {WrongSig, _}} ->
```

Comment:
I'm tempted to generalize further to `{ok, WeirdHeader}` or somesuch.

Reply:
I'd rather not generalize here as I don't like that we're getting `{ok, SomethingBad}` and I'd rather force our hand if it comes to that rather than just resetting and papering over something likely serious. The headers are all md5'ed so its got to be programmer error of some sort at that level.